### PR TITLE
Add react-semantic-toasts package for showing notifications

### PR DESCRIPTION
**Pull request by pradumangoyal**
We really need some good notification/toasts system at the frontend which looks like `Message` component of the semantic-ui. This functionality is not provided by the semantic-ui for now. So, we can use this package (https://www.npmjs.com/package/react-semantic-toasts). This PR must be merged with https://github.com/IMGIITRoorkee/omniport-backend/pull/62.

### CODE REVIEW

**Comment by pradumangoyal** [comment]
Demo: https://codesandbox.io/s/1wvqkq5qx7?from-embed

**Comment by liquidslr** [comment]
LGTM, @pradumangoyal  have you checked on how will these toast notifications can be used by other apps, will they also need to include the `<SemanticToastContainer />` separately in their app.js ?

File: omniport/package.json [hunk 1]
```diff
@@ -92,4 +93,4 @@
     "webpack-livereload-plugin": "^2.2.0",
     "webpack-merge": "^4.2.1"
   }
-}
+}
```
Comment (liquidslr) [inline omniport/package.json]: Add a new line at the end of the file

Comment (pradumangoyal) [reply omniport/package.json]: My linter automatically did this to me, Should I change it?

**Comment by pradumangoyal** [comment]
Yes, the only imported code in main App.js will work for every app.

File: omniport/package.json (hunk 1 above)
Comment (liquidslr) [reply omniport/package.json]: yes i guess, its recommended to leave a new line at the end of the file

**Comment by liquidslr** [comment]
I guess that will be an issue, we might need to enforce the same position for the notification for each app

**Pull request merged**